include/dem/data_containers.h

```diff
@@ -203,18 +203,20 @@ namespace dem_data_containers
       cells_total_neighbor_list;
 
     // <boundary id, (tensor, point)>
-    typedef std::map<types::boundary_id, std::pair<Tensor<1, 3>, Point<3>>>
+    typedef std::unordered_map<types::boundary_id,
+                               std::pair<Tensor<1, 3>, Point<3>>>
       boundary_points_and_normal_vectors;
 
     // <int, <boundary id, tensor>>
-    typedef std::map<unsigned int, std::map<types::boundary_id, Tensor<1, 3>>>
+    typedef std::unordered_map<unsigned int,
+                               std::map<types::boundary_id, Tensor<1, 3>>>
```

Comment:
Why not 2 unordered_map or unordered_map of vector?

Reply:
Fixed. still, this will continue to require refactoring maybe in a follow up PR. :).